modules/combined/test/tests/frictional_contact/single_point_2d/tests

```diff
@@ -1,23 +1,28 @@
 [Tests]
-  [./base]
+  issues = '#6524'
+  design = 'syntax/Contact/index.md MechanicalContactConstraint.md'
+  [./single_point_2d]
     type = 'Exodiff'
     input = 'single_point_2d.i'
     exodiff = 'single_point_2d_out.e'
     custom_cmp = 'single_point_2d.cmp'
     superlu = true
+    requirement = 'The contact system shall enforce 2D single-point contact with significant accumulated slip. '
   [../]
-  [./predictor]
+  [./single_point_2d_predictor]
     type = 'Exodiff'
     input = 'single_point_2d_predictor.i'
     exodiff = 'single_point_2d_predictor_out.e'
     custom_cmp = 'single_point_2d.cmp'
     superlu = true
+    requirement = 'The contact system shall enforce 2D single-point contact with significant accumulated slip. With predictor solver options. '
   [../]
-  [./tangential_penalty]
+  [./single_point_2d_tp]
     type = 'Exodiff'
     input = 'single_point_2d_tp.i'
     exodiff = 'single_point_2d_tp_out.e'
     custom_cmp = 'single_point_2d.cmp'
     superlu = true
+    requirement = 'The contact system shall enforce 2D single-point contact with significant accumulated slip when formulation selected is penalty contact. '
```

Comment:
penalty -> tangential_penalty
Also, remove the extra whitespace at the end of this requirement and the one above.